src/components/facet-browser/FacetBrowserModalBody.tsx

```diff
@@ -50,15 +40,16 @@ const FacetBrowserModalBody: React.FunctionComponent<
         // Remove facets disclosures with no tags
         if (values.length === 0) return null;
 
+        const hasChosenTerms = Boolean(filters[name]);
```

Comment:
Consider using the word "selected" instead of "chosen". It is also what we use for the individual terms on L58.